cekit/template_helper.py

```diff
@@ -88,6 +88,13 @@ def cachito(self, image):
         if image.get('osbs', {}).get('configuration', {}).get('container', {}).get('remote_source'):
             return True
 
+    def extra_dir(self, image):
+        return image.get('osbs', {}).get('extra_dir', {'osbs_extra'})
+
+    def extra_dir_target(self, image):
+        if image.get('osbs', {}).get('extra_dir_target', {}):
+            return image.get('osbs', {}).get('extra_dir_target', {})
```

Comment:
Why not just `return image.get('osbs', {}).get('extra_dir_target', {})` and removing the `if` entirely?

Reply:
Why the default value for `extra_dir_target` is set to `{}`?